### ISSUE TITLE
Sullep romboló

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#103]
```
A Sullep romboló ugyan úgy néz ki, mint az Ecalep romboló! Kérlek pótold 
az értékeit a Sullep rombolót a battle, illetve a tech.xml-be.
```

Original issue reported on code.google.com by `Jozsef.T...@gmail.com` on 4 Sep 2011 at 6:47

**Comment by GoogleCodeExporter** [comment]
```
Ezt hogy érted? Az eredetiben ugyanúgy néztek ki?

Jelenleg nincs sullep romboló, mert hiányzott hozzá az erőforrás. Van 
viszont 2 db Sullep vadász.
```

Original comment by `akarn...@gmail.com` on 4 Sep 2011 at 8:43

**Comment by GoogleCodeExporter** [comment]
```
Igen, eredetiben Ecalep romboló1 = Sullep Romboló1 (kinézetre persze). Ha 
nem gond, akkor amit e-mailben küldtem xml-t abba írd/rakd majd be.
```

Original comment by `Jozsef.T...@gmail.com` on 4 Sep 2011 at 8:46

**Comment by GoogleCodeExporter** [comment]
```
Fixed in SVN.
```

Original comment by `Jozsef.T...@gmail.com` on 4 Sep 2011 at 11:06
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `akarn...@gmail.com` on 5 Sep 2011 at 6:13
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed 0.93.450
```

Original comment by `akarn...@gmail.com` on 1 Oct 2011 at 12:38

**Comment by GoogleCodeExporter** [comment]
Original comment by `akarn...@gmail.com` on 1 Oct 2011 at 12:38
- Changed state: **Fixed**
